src/main/resources/spookytime.mixins.json

```diff
@@ -1,4 +1,15 @@
 {
+<<<<<<< HEAD
+    "required": true,
+    "package": "com.fabriccommunity.spookytime.mixin",
+    "compatibilityLevel": "JAVA_8",
+    "mixins": [
+        "LightmapTextureManagerMixin",
+	"AnvilBlockMixin",
+        "ServerWorldLightningMixin"
+    ],
+    "client": [
+=======
```

Comment:
Please grab the latest `spookytime.mixins.json` and replace this one, and add your own mixin to it. Something happened and causes this monstrosity.